### PR TITLE
[blend2d] Update to beta_2020-03-29

### PR DESCRIPTION
**Pull request by ZeeWanderer**
- Blend2D updated to beta_2020-03-29
- local Asmjit updated to 2020-03-16

### CODE REVIEW

File: ports/blend2d/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: blend2d
-Version: beta_2019-12-27
+Version: beta_2020-03-29
 Description: Beta 2D Vector Graphics Powered by a JIT Compiler
 Default-Features: jit, logging
```
Comment (NancyLi1013) [inline ports/blend2d/CONTROL]: It seems that `blend2d` currently doesn't support `arm `and `uwp`.
Could you please add `Supports: !(arm|uwp) `here?

File: ports/blend2d/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/blend2d/portfile.cmake]: include(vcpkg_common_functions) is no longer needed.
Could you please remove this line?

Comment (NancyLi1013) [reply ports/blend2d/portfile.cmake]: Could you please add `vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")` to the top of this file?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/blend2d/CONTROL (hunk 1 above)
Comment (ZeeWanderer) [reply ports/blend2d/CONTROL]: done

File: ports/blend2d/portfile.cmake (hunk 2 above)
Comment (ZeeWanderer) [reply ports/blend2d/portfile.cmake]: done

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Need to test features.

**Comment by ZeeWanderer** [comment]
Right now building without jit is experimental. Building without logging works fine.

**Comment by NancyLi1013** [comment]
@ZeeWanderer Thanks for your testing and updates.

LGTM, thanks for your PR.

All features have passed with the following triplets:
- x86-windows
- x64-windows
- x64-windows-static

**Comment by dan-shaw** [comment]
thanks for the PR!

**Pull request merged**